gooey/gui/model.py

```diff
@@ -207,8 +208,18 @@ def skipping_config(self):
     return self.build_spec['manual_start']
 
   def is_required_section_complete(self):
-    completed_values = filter(None, [arg.value for arg in self.required_args])

```

Comment:
Technically... _technically..._ you shouldn't be making UI changes here in the model and instead use the Presenter class, but, due to my many, many code organization sins, you'd probably have to change a whole lot of stuff in order to make that work. Further, all this code all changes in 2.0, so ultimately it fine living here